config/job.js

```diff
@@ -118,6 +118,13 @@ const SCHEMA_STEPS_NO_DUPS = Joi.array().items(SCHEMA_STEP).min(1).unique((a, b)
     return Object.keys(a).some(key => b[key]);
 });
 const SCHEMA_TEMPLATE = Joi.string().regex(Regex.FULL_TEMPLATE_NAME);
+const SCHEMA_TEMPLATE_DETAILS = Joi.object({
+    name: Joi.string(),
```

Comment:
you can refer to Template's schema, for example `Template.name`.
https://github.com/screwdriver-cd/data-schema/blob/3c990f2fbe314a6e34ad80342e135718256f22d4/config/template.js